mne/viz/misc.py

```diff
@@ -311,19 +312,19 @@ def _plot_mri_contours(mri_fname, surf_fnames, orientation='coronal',
         ax.axis('off')
 
         # and then plot the contours on top
-        for surf in surfs:
+        for surf, color in surfs:
             if orientation == 'coronal':
                 ax.tricontour(surf['rr'][:, 0], surf['rr'][:, 1],
                               surf['tris'], surf['rr'][:, 2],
-                              levels=[sl], colors='yellow', linewidths=2.0)
+                              levels=[sl], colors=color, linewidths=2.0)
             elif orientation == 'axial':
                 ax.tricontour(surf['rr'][:, 2], surf['rr'][:, 0],
                               surf['tris'], surf['rr'][:, 1],
-                              levels=[sl], colors='yellow', linewidths=2.0)
+                              levels=[sl], colors=color, linewidths=2.0)
             elif orientation == 'sagittal':
                 ax.tricontour(surf['rr'][:, 2], surf['rr'][:, 1],
                               surf['tris'], surf['rr'][:, 0],
-                              levels=[sl], colors='yellow', linewidths=2.0)
+                              levels=[sl], colors=color, linewidths=2.0)
```

Comment:
I find the flat code pretty readable, probably more readable than if I would have to look up indexing operations in a different place, but if a majority prefer I can refactor it